packages/block-editor/src/components/block-variation-transforms/style.scss

```diff
@@ -1,3 +1,10 @@
+fieldset.block-editor-block-variation-transforms {
```

Comment:
Is there a reason this one needs to be prefixed with `fieldset`? I'm guessing the classname applied to non-fieldset elements too.

Just wondering if we want to be careful to keep the specificity low on these resets.

Reply:
Oh, good question! Due to how the `block-editor-block-variation-transforms` classname is applied, it can be used in two different states for variation transforms, and we only want the `fieldset` state. I can give it a `:where()` to reduce the specificity back down.

Reply:
Good catch. I just manually tested it by forcing the buttons to display, and with a `:where()` thrown in, it doesn't override the other padding rule below:

<img width="726" height="878" alt="image" src="https://github.com/user-attachments/assets/4ee83716-5d3a-42e3-8109-f5aa04925382" />